### ISSUE TITLE
Accessing the logits during beam search

### ISSUE THREAD

**Issue by Rosenswagger** [tensorflow/nmt#318]
I'd like to view the relative probabilities for each inference made through beam search. I think this is as simple as a softmax of the logits at each step, but I can't figure out where to implement this. Has anyone tried this?

Ex: Instead of output:
1st Beam Option
2nd Beam Option
3rd Beam Option

I'd like:
1st Beam Option (60%)
2nd Beam Option (20%)
3rd Beam Option (10%)
